styles.css

```diff
@@ -145,3 +145,320 @@ h2 {
+  background-color: white;
+  color: #396df2;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 24px;
+  padding-left: 2px;
+  cursor: pointer;
+  width: 115px;
+  display: flex;
+}
+
+.about-myself-link:hover {
+  color: #fff;
+  background-color: #7f8cff;
+  border-radius: 5px;
+}
+
+.about-myself {
+  display: grid;
+  width: 350px;
+  padding: 15px;
+  border-radius: 20px;
+  align-items: center;
+  margin-bottom: 100px;
+  background-color: white;
+  height: 1100px;
+}
+
+.about-myself h1 {
+  width: 100px;
+  height: 50px;
+  margin-top: 50px;
+  color: #172b4d;
+}
+
+.about-myself-icons > img {
+  width: 14px;
+  padding: 7px;
+  height: 15px;
+}
+
+.resume {
+  border: 1px solid #396df2;
+  background-color: #fff;
+  width: 111px;
+  height: 30px;
+  border-radius: 4px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #396df2;
+  font-size: 15px;
+}
+
+.resume:hover {
+  background-color: #6070ff;
+  cursor: pointer;
+  color: #fff;
+}
+
+.about-myself-ul li {
+  display: flex;
+  align-items: center;
+  border: 1px solid white;
+  border-radius: 5px;
+  background-color: #f7f7f9;
+  height: 62px;
+  padding-left: 8px;
+}
+
+.about-myself-ul {
+  padding-left: 2px;
+  gap: 12px;
+  display: flex;
+  flex-direction: column;
+  font-size: 14px;
+}
+
+.about-myself-ul span {
+  margin-left: 15px;
+}
+
+.frameworks li {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.frameworks img {
+  margin-right: 50px;
+  color: #253858;
+}
+
+.skills li {
+  display: grid;
+  grid-template-columns: auto auto;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.skills img {
+  margin-right: 50px;
+  color: #253858;
+}
+
+.header-text p {
+  color: #344563;
+}
+
+.language {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  border: none;
+  font-size: 18px;
+  color: #396df2;
+  border-radius: 6px;
+}
+
+.language img {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  margin-right: 50px;
+}
+
+.language:hover {
+  color: #fff;
+  background-color: #6070ff;
+}
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  background-image: url(./photo/bg/contact\ form\ shapes-mobile@2x.png);
+  width: 350px;
+  height: 700px;
+  padding: 15px;
+  border-radius: 20px;
+  border: 2px solid white;
+  align-items: center;
+  margin-bottom: 100px;
+  background-color: #6070ff;
+  margin-top: 50px;
+}
+
+.form-container .form-text h1 {
+  font-family: 'Arial Narrow', 'poppins', Arial, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex-direction: column;
+  margin-top: 80px;
+}
+
+.form-container .paragragh p {
+  padding-left: 2px;
+  padding-right: 35px;
+  width: 300px;
+  font-family: 'Arial Narrow', Arial, sans-serif, 'poppins';
```

Comment:
[Optional]
- [ ] It's better if you follow the `font-family` specified by the Figma design (which is Poppins), it's also better if you follow same `font-size` to make your project look a replica of the design 💯